addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -262,6 +262,10 @@ def _iap_enrich(self, domain):
         Returns enrichment data for a given domain, in case an error happens the response will
         contain an enrichment_info key explaining what went wrong
         """
+        if domain in iap_tools._MAIL_DOMAIN_BLACKLIST:
+            # Can not enrich the provider domain names (gmail.com; outlook.com, etc)
+            return {'enrichment_info': {'type': 'missing_data'}}
```

Comment:
Shouldn't we have 'info' key also ?

Reply:
no, only `enrichment_info` is needed here (the addin parse the error code to show the right error message)